samples/database-create-with-version-retention-period.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright 2020 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+async function createDatabaseWithVersionRetentionPeriod(
+  instanceId,
+  databaseId,
+  projectId
+) {
+  // [START spanner_create_database_with_version_retention_period]
+  // Imports the Google Cloud client library
+  const {Spanner} = require('@google-cloud/spanner');
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const projectId = 'my-project-id';
+  // const instanceId = 'my-instance';
+
+  // Creates a client
+  const spanner = new Spanner({
+    projectId: projectId,
+  });
+
+  const instance = spanner.instance(instanceId);
+  const database = instance.database(databaseId);
+
+  try {
+    // Creates a new database.
+    console.log(`Creating database ${instance.formattedName_}.`);
+    const versionRetentionStatement = `
```

Comment:
nit: I think we should add a short comment here on the fact that the `database.create()` method does not support specifying a retention period, but that we execute it as an extra `ALTER DATABASE` statement after the database has been created. (I had to read the code a couple of times before I really saw that, as the sample and function is called `createDatabaseWithVersionRetentionPeriod`).

Reply:
Yes, completely agree that this is not obvious. I added a comment to describe what's happening. Thanks for the suggestion.